tests/test_browsing_context.py

```diff
@@ -956,3 +956,32 @@ async def test_browsingContext_fragmentNavigated_event(websocket, context_id,
 
     assert navigation['result']['navigation'] == response['params'][
         'navigation']
+
+
+@pytest.mark.asyncio
+async def test_browsingContext_navigationStarted_event(websocket, context_id,
+                                                       html):
+    url = html()
+
+    await subscribe(websocket, ["browsingContext.navigationStarted"])
```

Comment:
you can drop the brackets as we're passing a single event

Reply:
I use it as it more closely resembles BiDi, happy to change if there is already agreed way for this.

Reply:
then it would be nicer to drop the string option, in order to make it unambiguous w.r.t. style (but then it's out of scope of this PR)